File: CSharp.Functions/Common/Entities/FirmDetails/Locum/LocumDetails.cs
namespace Common.Entities
{
    public class LocumDetails
    {
        public bool IsDependentOnSingleKeyIndividual { get; set; }
        public bool? IsLocumCompany { get; set; }
        public bool? HasAppointedALocum { get; set; }
        public string? HasNoAppointedLocumInfo { get; set; }
        public LocumCompanyDetails? LocumCompanyDetail { get; set; }
        public LocumSoleTraderDetails? LocumSoleTraderDetail { get; set; }
    }
}

File: vue.jsonforms/src/entities/LocumDetails.ts
import { LocumSoleTraderDetails } from "@/entities/firm-details/LocumSoleTraderDetails";
import { LocumCompanyDetails } from "@/entities/firm-details/LocumCompanyDetails";

export class LocumDetails {
  public isDependentOnSingleKeyIndividual?: boolean;
  public isLocumCompany?: boolean;
  public hasAppointedALocum?: boolean;
  public hasNoAppointedLocumInfo?: string;
  public locumCompanyDetail?: LocumCompanyDetails;
  public locumSoleTraderDetail?: LocumSoleTraderDetails;
}